api/weeklyPicks.test.tsx

```diff
@@ -0,0 +1,18 @@
+test('get weekly picks mock function', async () => {
```

Comment:
This test file needs to match the name of the file it came from. authFunctions.test.ts